views/tx.tmpl

```diff
@@ -93,7 +93,7 @@
                         <!-- <td>{{$vin.Tree}}</td> -->
                         <td>
                         {{if eq $vin.BlockHeight 0}}
-                            N/A
```

Comment:
This is generally referred to as either "unconfirmed" or "in mempool".

@raedah and @gozart1 Any preferences here?

Reply:
unconfirmed sounds good

Reply:
unconfirmed

Reply:
unconfirmed sounds good to me

Reply:
Actually the 0 in this case is referring to a Vote Reward where the coins are created for the first time and have no input block height. So I had suggested 'Created'. Could also be mined or coinbase.

Reply:
oh, I see this is the "In" table.  My bad.  Yes, since the coins are generate this is input does not actually have a previous outpoint at all.  However, the "Previous Outpoint" column says something like "Coinbase: 00002f646372642f".  For example:  https://beta.dcrdata.org/explorer/tx/941505dd59ff29bebfa986c5192fcf93e86a10732df26e32643809345eae3ddd

So showing N/A does seem appropriate for the "Block" column, although blank would be good too.  Not sure created makes sense to indicate the origin block for the coinbase input.

Reply:
Generated coins do not have a origin block. There is no source height because the coins did not previously exist. N/A suggests that its not available for some reason, like not yet being confirmed, but its actually not even logical. Leaving it blank just looks like a bug where data is missing. Both of those a liable to confuse and mislead the user, and neither of those are an actual block height either. If Created or Generated is not clear enough, more verbose would be (Newly Generated Coins). In the same way that Vote Reward is not a previous outpoint, Created is also not a block. Instead we are accurately informing the user of what is occurring.

Reply:
N/A suggests that its not available for some reason, like not yet being confirmed, but its actually not even logical. Some users may not know what a Coinbase or Vote Reward is and could use additional description. In the same way that a Coinbase or Vote Reward is not a previous outpoint, Created is also not a block. The Block column is just the right size to put the word Created which informs the user of what is occurring, and can be used for both Coinbase and Vote Reward special cases.

Reply:
2000 for addressRows was a bit much, so I finally decided to cut it down to 700 to save a bit of load time.

Reply:
just to reduce review iteration time, a screenshot is sometimes helpful  :)

Reply:
So this is still not right because for regular (non-stake) unconfirmed transactions, it will show "Created" in the Block column for the inputs.  There is an issue with having no input values for unconfirmed transactions, which we do need to solve, but in the meantime this check should not be `{{if eq $vin.BlockHeight 0}}` but the same logic that was used for the Previous Outpoint column:

```
{{if $vin.Coinbase}}
    Coinbase: {{ $vin.Coinbase }}
{{else if eq $vin.Txid "0000000000000000000000000000000000000000000000000000000000000000"}}
    Vote Reward
{{else}}
...
```

except we'll need to OR the first two conditions